src/test/data/vcf-test.js

```diff
@@ -51,3 +52,58 @@ describe('VCF', function() {
     });
   });
 });
+describe('VCF from string content', function() {
```

Comment:
Indeed, it's better.

For my defence I will only say that this was the first thing that I wrote in your project and I didn't know a lot about the code that you have, and didn't think later on about refactoring it...